nucypher/characters/lawful.py

```diff
@@ -1109,7 +1109,10 @@ def run(self,
             pruning: bool = True,
             interactive: bool = False,
             prometheus: bool = False,
-            start_reactor: bool = True
+            start_reactor: bool = True,
+            metrics_port: int = None,
+            metrics_prefix: str = 'ursula',
+            metrics_listen_address: str = ''
```

Comment:
These values are okay here for now but can be better encapsulated I think.

Reply:
I think it should be another PR with full refactoring of some functions